src/System.Management.Automation/utils/Verbs.cs

```diff
@@ -1123,34 +1123,22 @@ public class VerbInfo
         /// <summary>
         /// The verb name, used to begin command names.
         /// </summary>
-        public string Verb
-        {
-            get; set;
-        }
+        public string Verb { get; internal set; }
```

Comment:
Maybe make the default constructor internal too?

Reply:
Made the default constructor internal.